node-operators/rpc.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Run an RPC Node Using Docker
+description: How to set up and run a ContainerChain node using Docker, so you can have your own RPC endpoint-as-a-backend to support your DApps and interact with your chain.
+---
+
+# Run an RPC Node Using Docker
+
+## Introduction {: #introduction }
+
+Running an RPC node on a Tanssi ContainerChain allows you to connect to the network, discover other nodes to synch with through a bootnode, and offer an interaction layer for DApps over the WebSocket protocol.
+
+In this guide, you'll learn how to quickly spin up a ContainerChain node using [Docker](https://www.docker.com/){target=\_blank} on a Linux computer.
+
+!!! note
+    It is not possible to run an RPC node for Snap Appchains as they run on a private network, and their nodes are, therefore, unable to reach for synching.
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To get started, you'll need access to a computer running a Linux OS and install [Docker](https://docs.docker.com/desktop/install/linux-install/){target=\blank}.
+
+Run the following command to install Docker on a Linux Ubuntu platform:
+
+```bash
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+
+And the following command to check the installation:
+
+```bash
+sudo docker run hello-world
+```
+ 
+This is what a successful execution in the terminal looks like:
+
+--8<-- 'code/node-operators/rpc/terminal/hello-world.md'
+
+## Pulling the Docker Image {: #pulling-docker-image }
```

Comment:
Might be helpful to have the EVM image and Substrate image in `###` headings so that people can get to it from the navigation menu more directly

Reply:
Also consider what happens when you have your own custom runtime. What do you do then?

Reply:
Good one! I guess for custom specs we are building another image, I'll look into that

Reply:
(Custom specs are classified as either evm or non evm, therefore they are bundled in the same images as the evm/substrate templates)

Reply:
EVM capitalized